config/requests.yml.tmpl

```diff
@@ -0,0 +1,14 @@
+# requests.yml
+
+defaults: &defaults
+  umlaut_base: https://getit.princeton.edu
+  bibdata_base: http://bibdata.princeton.edu/bibliographic
```

Comment:
Should any code that uses `http://bibdata.princeton.edu/locations` be refactored to use config/constants from this file? Or is that all Javascript? js.erb? or is that bad practice?